setup.py

```diff
@@ -206,7 +206,7 @@ def do_setup():
             'configparser>=3.5.0, <3.6.0',
             'croniter>=0.3.17, <0.4',
             'dill>=0.2.2, <0.3',
-            'flask>=0.11, <0.12',
+            'flask>=0.12, <0.13',
```

Comment:
Need to add flask-appbuilder in here too somewhere don't we?

Reply:
oops